build.gradle

```diff
@@ -42,9 +44,11 @@ subprojects {
   apply plugin: 'nebula.source-jar'
   apply plugin: 'java'
   apply plugin: 'maven' // make pom files for deployment
+  apply plugin: 'nebula.maven-base-publish'
 
   repositories {
     mavenCentral()
+    maven { url "http://palantir.bintray.com/releases" }
```

Comment:
Was this included by accident? I don't think the open source project should publish to a Palantir repository, right?

Reply:
This is to pick up the palantir-git-version plugin.

Reply:
Actually never mind this doesn't apply here because the plugin is in the buildscript. And clearly the build passes so we don't need to get it from bintray.

Reply:
Oh right, this is configuring dependency repositories, not a deployment. My bad.